### PR TITLE
Enhancement/596 piped script

### PR DESCRIPTION
**Pull request by DnlLrssn**
**Checklist**

- [ ] tests added
- [x] commits conform to the [Commit message format](https://github.com/qlik-oss/gopherciser/blob/master/.github/CONTRIBUTING.md#commit)
- [x] documentation updated

**Squash commit message**

- enable piping cfgfile from stdin
- enable piping overrides from stdin

**Info**

Optional informative text (not to be included in commit message) relevant to reviewers.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -181,6 +181,22 @@ Sub-commands:
 * `-f`, `--force`: Overwrite existing scenario setup file.
 * `-h`, `--help`: Show the help for the `template` command.
 
+##### Piping stdin
+
+Config file and override file can be piped from stdin. If no config is set stdin is assumed to be the config file, if config file is set, stdin is assumed to be the ovverides file.
```
Comment (markuswall) [inline docs/README.md:186]: Tiny spelling error: ovverrides

Comment (SeBBBe) [reply docs/README.md:186]: Also it's called "override file" in the previous sentence

**Review by SeBBBe — APPROVED** [review]
Cool stuff!

File: docs/README.md (hunk 1 above)
Comment (DnlLrssn) [reply docs/README.md]: should be fixed now

**Review by SeBBBe — APPROVED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -181,6 +181,22 @@ Sub-commands:
...
+This would execute the sheetchanger example from stdin:
+
+```bash
+cat ./docs/examples/sheetChangerQlikCore.json | ./gopherciser x
+```
+
+This would execute overrides from stdin:
+
+```bash
+cat overrides.txt |./gopherciser x -c ./docs/examples/sheetChangerQlikCore.json
```
Comment (atluq) [inline docs/README.md:197]: nit: add a space
```suggestion
cat overrides.txt | ./gopherciser x -c ./docs/examples/sheetChangerQlikCore.json
```

**Review by atluq — APPROVED** [review]
Nice feature to have!

File: docs/README.md (hunk 2 above)
Comment (DnlLrssn) [reply docs/README.md:197]: why?

Comment (SeBBBe) [reply docs/README.md:197]: fwiw I agree with Anton

Comment (atluq) [reply docs/README.md:197]: semantically the same, but it is the norm to surround with spaces from what I know

Comment (DnlLrssn) [reply docs/README.md:197]: really? my term emulator always automatically remove the space from there

**Pull request merged**